DigitalLearningSolutions.Web/Controllers/RegisterController.cs

```diff
@@ -128,8 +130,10 @@ public IActionResult Password(PasswordViewModel model)
         [HttpGet]
         public IActionResult Summary()
         {
-            var data = TempData.Peek<DelegateRegistrationData>();
-            var viewModel = MapToSummary(data!);
+            var data = TempData.Peek<DelegateRegistrationData>()!;
+            var centre = centresDataService.GetCentreName((int)data.LearnerInformationViewModel.Centre!);
+            var jobGroup = jobGroupsDataService.GetJobGroupName((int)data.LearnerInformationViewModel.JobGroup!);
+            var viewModel = MappingHelper.MapToSummary(data, centre!, jobGroup!);

```

Comment:
I wonder whether the MappingHelper should be renamed to RegistrationMappingHelper to be more specific..

Reply:
It depends on whether we want to make a new Helper class if another controller needs to do similar things in the future or if we put them all in here. I sort of assumed the second (since it's a static class anyway) but happy to rename it if you want.

Reply:
I'm worried that this class will get too big and hard to scan over if we just have one mapping helper per project (this has happened on previous projects I've worked on so keen to avoid it from the start)